_aggregations/metric/geocentroid.md

```diff
@@ -142,14 +226,15 @@ PUT /locations
   }
 }
 
-POST /locations/_bulk?refresh
+POST /locations/_bulk
 {"index": {"_id": 1}}
-{"name": "Point A", "coordinates": ["40.7128, -74.0059", "51.5074, -0.1278"]}
+{"name": "Set A", "coordinates": ["40.7128, -74.0059", "51.5074, -0.1278"]}
```

Comment:
I find this confusing. There are 2 pairs of lat, lon so what exactly does this represent?
I would suggest removing this example.

Reply:
@bowenlan-amzn As far as I know, it's just an arbitrary set of points to illustrate the fact that a document can end up in multiple buckets. Could be more than two points, that's just the simplest example. Because each one contains multiple points, I found calling the document "Point" confusing, which is why I renamed them "Set".

I think we should clarify the example if it's confusing, but I'm hesitant to remove it. This concept (that a centroid is computed on all of a documents points for each bucket) is important and could lead to some bad calculations if misunderstood.

Reply:
We better have a good meaningful use case here to illustrate. Otherwise, we can just mention geo_point field support multi values in geo_point doc page.
For how aggregation process multi-values document, we can probably explain in the main doc page for aggregation. somewhere like https://opensearch.org/docs/latest/aggregations/#general-aggregation-structure

Reply:
@bowenlan-amzn Are you suggesting replacing this example? Or removing it? I think it's the simplest example that illustrates the concept.

I also think adding a reference to the Aggregations intro page is a good idea if it helps users figure out how nested aggregations work.

Reply:
I think this example only helps show geo_point field support multi values. 
I don't like the example putting here because it contains `geohash_grid` as the first aggregation, but this doc is only for `geo_centroid` agg. So I suggest move it to `geo_point` or `geohash_grid` page.

Reply:
OK, I'll look at this again and see if the example is really necessary. Or if I can come up with a better one.

Reply:
@dwelsch-esi any update here? I think at least this should be moved to other places.

Reply:
@bowenlan-amzn OK, I looked at it again, and you've convinced me the example is unnecessary for the geocentroid page. It might be a better fit for the geohash page. Deleting from here for now.